package.json

```diff
@@ -16,6 +16,8 @@
     "angularjs-dragula": "2.0.0",
     "dompurify": "1.0.9",
     "esn-frontend-common-libs": "github:openpaas-suite/esn-frontend-common-libs#main",
+    "esn-frontend-inbox-calendar": "github:openpaas-suite/esn-frontend-inbox-calendar#main",
+    "esn-frontend-mailto-handler": "github:openpaas-suite/esn-frontend-mailto-handler#main",
```

Comment:
No, this one is used outside of inbox, we are in inbox, a mailto link will open the inbox composer, not the SPA

Reply:
Reverted this